t/sanity.t

```diff
@@ -647,3 +647,30 @@ GET /test
 nil
 --- no_error_log
 [error]
+
+=== TEST 19: upstream_name with valid explicit upstream set by variable
```

Comment:
Point is to test situation where current_upstream_name() is called with block of upstream set by variable. For example, current_upstream_name() works fine in issue 
https://github.com/openresty/lua-upstream-nginx-module/issues/39 when scheme is explicitly set (location: /test). And returns error when scheme is set via variable (location: /test2).